wfe2/wfe.go

```diff
@@ -810,21 +780,13 @@ func (wfe *WebFrontEndImpl) NewAccount(
 		InitialIP:       ipBytes,
 	}
 
-	// TODO(#5545): Spending and Refunding can be async until these rate limits
-	// are authoritative. This saves us from adding latency to each request.
-	// Goroutines spun out below will respect a context deadline set by the
-	// ratelimits package and cannot be prematurely canceled by the requester.
-	txns := wfe.newNewAccountLimitTransactions(ip)
-	go wfe.checkNewAccountLimits(ctx, txns)
+	refundLimits := wfe.checkNewAccountLimits(ctx, ip)
 
 	var newRegistrationSuccessful bool
 	var errIsRateLimit bool
 	defer func() {
```

Comment:
Thanks, this totally makes sense here. However, I'm going to keep it the same because we cannot do the same thing in NewOrder due to how we increment metrics there.